docs/diagrams/CompletionRateSequenceDiagram.puml

```diff
@@ -1,9 +1,9 @@
 @startuml
-actor User as user
-participant Ui
-participant Hospital
-participant Patient
-participant TaskList
+actor ":User" as user
```

Comment:
Should add the color codes for the sequence diagrams based on the ones in style.puml for standardisation